src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -165,6 +171,46 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
     throw new Error("Foundation phase is not supported");
   }
 
+  const pupilExitQuiz = sectionResults["exit-quiz"]?.questionResults
+    ? sectionResults["exit-quiz"]?.questionResults.map((result, index) => {
+        return {
+          pupilExperienceLessonActivity:
+            "exit-quiz" as PupilExperienceLessonActivityValueType,
```

Comment:
Type casting shouldn't be necessary here. Looks like you might need to define your return type properly